chapters/ch38.md

```diff
@@ -1,35 +1,31 @@
 
 > How is this New York City vs. Detroit example different from the mobile app vs. internet cat images example?
 
---> _replace THIS LINE by your translation for the above line_
-
+Ví dụ về thành phố New York và Detroit này khác gì so với ví dụ về ảnh mèo từ ứng dụng di động và ảnh trên internet?
 
 > The cat image example is different because, given an input picture x, one can reliably predict the label y indicating whether there is a cat, even without knowing if the image is an internet image or a mobile app image. I.e., there is a function f(x) that reliably maps from the input x to the target output y, even without knowing the origin of x. Thus, the task of recognition from internet images is “consistent” with the task of recognition from mobile app images. This means there was little downside (other than computational cost) to including all the data, and some possible significant upside. In contrast, New York City and Detroit, Michigan data are not consistent. Given the same x (size of house), the price is very different depending on where the house is.
 
---> _replace THIS LINE by your translation for the above line_
-
+Ví dụ về ảnh mèo khác trường hợp trên bởi vì, với một ảnh đầu vào x, ta có thể dự đoán một cách đáng tin cậy nhãn y liệu có mèo trong ảnh hay không mà không cần biết hình ảnh đó là từ internet hay từ ứng dụng di động. Nghĩa là, có một hàm f(x) ánh xạ đáng tin cậy từ đầu vào x đến đầu ra mục tiêu y, ngay cả khi không biết nguồn gốc của x. Do đó, nhiệm vụ nhận dạng hình ảnh từ internet là nhiệm vụ thống nhất với nhiệm vụ nhận dạng hình ảnh từ ứng dụng di động. Điều này có nghĩa là có rất ít nhược điểm (ngoài chi phí tính toán) khi bao gồm tất cả các dữ liệu và một số nhược điểm đáng kể có thể có. Ngược lại, dữ liệu của thành phố New York và Detroit, Michigan không nhất quán. Cho cùng một x (kích thước của ngôi nhà), giá nhà rất khác nhau tùy thuộc vào nơi nhà.
 
 > **FOOTNOTE:**
 
+**CHÚ THÍCH:**
+
 > [13] There is one way to address the problem of Detroit data being inconsistent with New York City data, which is to add an extra feature to each training example indicating the city. Given an input x—which now specifies the city—the target value of y is now unambiguous. However, in practice I do not see this done frequently.
 
---> _replace THIS LINE by your translation for the above line_
+[13] Có một cách để giải quyết vấn đề dữ liệu Detroit không nhất quán với dữ liệu của Thành phố New York, đó là thêm một tính năng bổ sung cho mỗi ví dụ huấn luyện. Với một đầu vào x hướng cụ thể về một thành phố, giá trị mục tiêu của y hiện không rõ ràng. Tuy nhiên, trong thực tế tôi không thấy điều này được thực hiện thường xuyên.
```

Comment:
Đoạn gốc: `which is to add an extra feature to each training example indicating the city`.
Đoạn bạn dịch: `đó là thêm một tính năng bổ sung cho mỗi ví dụ huấn luyện`.
Theo mình cần dịch lại `feature` --> `đặc trưng`, `example` --> `mẫu`. Bạn bị thiếu ý của vế `indicating the city`.
Dịch lại: `đó là thêm một đặc trưng vào mỗi mẫu huấn luyện trường ảm chỉ thành phố`.
Given an input x—which now specifies the city—the target value of y is now unambiguous.` nên được dịch là:
`Với một đầu vào x — mà đã xác định thành phố — giá trị của biến mục tiêu y bây giờ là rõ ràng.`

Reply:
Mình đã update lại rồi bạn 
"Có một cách để giải quyết vấn đề dữ liệu Detroit không nhất quán với dữ liệu của Thành phố New York, đó là thêm một đặc trưng biểu diễn thành phố. Cho một đầu vào x—với đặc trưng biểu diễn thành phố—giá trị mục tiêu của y bây giờ không mập mờ nữa."